index.ios.js

```diff
@@ -29,27 +18,18 @@ const store = createStore(
   applyMiddleware(thunk, promise, logger)
 );
 
-class findAR extends Component {
-
-  constructor() {
-    super();
-  }
+const scenes = routerActions.create(
+  <Scene key="root" hideNavBar>
+    <Scene key="ar" component={ARView} />
+    <Scene initial key="map" component={MapView} />
+    <Scene key="list" component={PinListView} />
```

Comment:
@jake-shasteen I agree! I just wanted to note these down so everyone is on the same page as to what we have to also do next with regards to routing